apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.test.js

```diff
@@ -0,0 +1,650 @@
+        expect(countWithoutPermissions).toBe(0)
+        const propertyWithoutPermissions = await Organization.getOne(serviceUserClient, { id: organization.id })
+        expect(propertyWithoutPermissions).toBeUndefined()
+
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await createTestOrganization(serviceUserClient)
+        })
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await updateTestOrganization(serviceUserClient, organization.id, {})
+        })
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await Organization.softDelete(serviceUserClient, organization.id)
+        })
+
+        const countByUser = await Organization.count(user, {})
+        expect(countByUser).toBe(1)
+
+        // you cannot create a field "canManageOrganizations: true". Only reading! Always false!
+        await catchErrorFrom(async () => {
+            await createTestB2BAppAccessRightSet(admin, right.id, {
+                canManageOrganizations: false,
+            })
+        }, (e) => {
+            expect(e.errors[0].message).toContain('Field "canManageOrganizations" is not defined by type "B2BAppAccessRightCreateInput"')
+        })
+
+        // add permissions for B2BApp
+        const [accessRightSet] = await createTestB2BAppAccessRightSet(admin, app, {
+            canReadOrganizations: true,
+        })
+        await updateTestB2BAppAccessRight(admin, right.id, { accessRightSet: { connect: { id: accessRightSet.id } } })
+
+        // you can update 'canReadOrganizations'
+        await updateTestB2BAppAccessRightSet(admin, accessRightSet.id, {
+            canReadOrganizations: true,
+        })
+
+        // you cannot update a field "canManageOrganizations". Only reading! Always false!
+        await catchErrorFrom(async () => {
+            await updateTestB2BAppAccessRightSet(admin, accessRightSet.id, {
+                canManageOrganizations: false,
+            })
+        }, (e) => {
+            expect(e.errors[0].message).toContain('Field "canManageOrganizations" is not defined by type "B2BAppAccessRightUpdateInput"')
+        })
+
+        // B2BApp with permission "canReadOrganization"
+        const countWithPermissions = await Organization.count(serviceUserClient, {})
+        expect(countWithPermissions).toBe(1)
+        const organizationWithPermissions = await Organization.getOne(serviceUserClient, { id: organization.id })
+        expect(organizationWithPermissions.id).toBe(organization.id)
+
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await createTestOrganization(serviceUserClient)
+        })
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await updateTestOrganization(serviceUserClient, organization.id, {})
+        })
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await Organization.softDelete(serviceUserClient, organization.id)
+        })
```

Comment:
I see 2 out of 3 cases here, it would be cool to add the third one

- [x] Have no accessRightSet
- [x] Have true permission
- [ ] Have false permissions

Reply:
done